app/services/home.js

```diff
@@ -15,10 +22,58 @@ class HomeService {
     }
 
     /**
-     * Lists all
+     * Handele upload of slider images
+     */
+    uploadSliderImages(req, res) {
+        if (Object.keys(req.files).length === 0) {
+            this.logger.error('No slider was selected for upload');
+            throw new errors.NoSliderSelected('No slider was selected for upload');
+        }
+        if (req.files.hasOwnProperty('slider')) {
+            let slider = req.files.slider;
+            let slider_id = req.query.id;
+            slider.mv('public/sliders/' + slider.name, err => {
+                if (err) {
+                    this.logger.error('Failed to move slider');
+                    throw err;
+                }
+            });
+            return new Slider().save({ image_url: '/sliders/' + slider.name, created_at: new Date(), updated_at: new Date(), slider_id }, { method: 'insert' })
```

Comment:
afaik, timeStamp columns don't need to be explicitly provided, bookshelf takes care of those

Reply:
i also expect this operation to be an update instead of an insert

The idea is that  we will pre populate the sliders table with 3 dummy rows by default. ( we need to create a seed migration for that )

Then this `updloadSliderImages` method will always be an update to change the sliders to the new image that users choose.